net/download/service/src/main/java/de/danoeh/antennapod/net/download/service/feed/FeedUpdateManagerImpl.java

```diff
@@ -111,6 +111,9 @@ private void confirmMobileRefresh(final Context context, @Nullable Feed feed) {
                     UserPreferences.setAllowMobileFeedRefresh(true);
                     runOnce(context, feed);
                 })
+                .setOnCancelListener((d) -> {
+                    EventBus.getDefault().postSticky(new FeedUpdateRunningEvent(false));
```

Comment:
Is this actually correct? Eg if some other podcast is still refreshing, this would disable the indicator even though there is actually still a refresh going on